assets/js/src/admin.js

```diff
@@ -26,3 +26,120 @@
 		} );
 	} );
 } )( window, jQuery );
+
+(function (window, $) {
+	'use strict';
+
+	var RSADisablePlugin = {
+
+		els: {
+			dialog: document.getElementById( 'rsa-disable-dialog' ),
+			placeholderA: document.getElementById( 'rsa-operator-a' ),
+			placeholderB: document.getElementById( 'rsa-operator-b' )
+		},
+
+		variables: {
+			operatorA: 0,
+			operatorB: 0,
+			expectedAnswer: 0,
+			disablingURL: null
+		},
+
+		openDialog: function ( event ) {
+			event.preventDefault();
+			$( this.els.dialog ).dialog( 'open' );
+		},
+
+		getRandomNum: function () {
+			return Math.floor( Math.random() * 10 );
+		},
+
+		isExpectedAnswer: function () {
+			var userResult = parseInt( document.getElementById( 'rsa-user-result' ).value, 10 );
```

Comment:
cache the 'rsa-user-result' selector since isExpectedAnswer might be called more than once

Reply:
Every time a user opens the dialog, new values are displayed, even on same page load. The idea is to make the "game" harder to solve. If we cache the result, we will end up in a situation where we'll be unable to check properly.